bunsen/bunsen-avro/src/test/java/com/cerner/bunsen/avro/AvroConverterMergeTest.java

```diff
@@ -107,18 +107,17 @@ public void validateMergedStu3UsCoreSchemas() throws ProfileException, IOExcepti
   public void validateR4UsCoreResourceWithExtension() throws ProfileException, IOException {
     FhirContext fhirContext =
         ProfileMapperFhirContexts.getInstance()
-            .contextFromClasspathFor(FhirVersionEnum.R4, "/r4-us-core-definitions");
+            .contextFor(FhirVersionEnum.R4, "classpath:/r4-us-core-definitions");
 
+    // We keep the fullId such that after HAPI->Avro->HAPI conversion, we get the same object.
```

Comment:
nit: This comment is not valid anymore I guess (after the removal of fullId)

Reply:
Done.